config/settings.yml

```diff
@@ -144,6 +144,9 @@
         :amqp_vhost: '/'
         :ceilometer:
           :event_types_regex: '\A(?!firewall|floatingip|gateway|net|port|router|subnet|security_group|vpn|volume|snapshot|backup)'
+        :stf:
+          :topic_name: anycast/ceilometer/event.sample
```

Comment:
Is the topic name really `event.sample?`

Reply:
Yes, it is. (the topic name it has been changed recently on OSP side, but the suffix ```event.sample``` is there..)